### PR TITLE
wallet: unit tests

### PR DESCRIPTION
**Pull request by volekerb**
### Problem
Week coverage
...

### Solution
This patchset increases coverage for wallet package
...

### CODE REVIEW

File: pkg/wallet/wallet_test.go [hunk 1]
```diff
@@ -1,8 +1,92 @@
...
+
+	path := wallet.Path()
+	assert.Equal(t, path, walletPath)
+}
+
+func TestSave(t *testing.T)  {
+	wallet := checkWalletConstructor(t)
+
+	err := wallet.Save()
+	assert.NoError(t, err)
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/wallet/wallet_test.go]: I think it should be symmetrical to `Save`, if `Save` outputs some kind of JSON into the file, then `Load` should read it.

**Comment by volekerb** [comment]
Codecov is fun -- added unit tests -> coverage decreases. Don't add tests -> coverage increased =)

File: pkg/wallet/wallet_test.go [hunk 2]
```diff
@@ -1,8 +1,103 @@
...
+	"io/ioutil"
+	"os"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var (
+	walletPath = "testWallet"
```
Comment (roman-khimov) [inline pkg/wallet/wallet_test.go]: I think now it can be a `const`, not `var.

File: pkg/wallet/wallet_test.go [hunk 3]
```diff
@@ -1,8 +1,103 @@
...
+	file, err := ioutil.TempFile("", walletPath)
+	assert.NoError(t, err)
+	wallet, err := NewWallet(file.Name())
+	defer removeWallet(t, file.Name())
+	require.NoError(t, err)
+	return wallet
+}
+
+func removeWallet(t *testing.T, walletPath string) {
+	err := os.RemoveAll(os.TempDir() + "/" + walletPath)
```
Comment (roman-khimov) [inline pkg/wallet/wallet_test.go]: Why do you add a `TempDir()` here? `file.Name()` already returns an absolute path to your file, here you're deleting something like `/tmp/tmp/testWalletXXXXX`.

File: pkg/wallet/wallet_test.go [hunk 4]
```diff
@@ -1,8 +1,103 @@
...
+
+	path := wallet.Path()
+	assert.NotEmpty(t, path)
+}
+
+func TestSave(t *testing.T) {
+	wallet := checkWalletConstructor(t)
+
+	err := wallet.Save()
+	assert.NoError(t, err)
```
Comment (roman-khimov) [inline pkg/wallet/wallet_test.go]: So, there should be some positive `NewWalletFromFile()` call there or not?

File: pkg/wallet/wallet_test.go (hunk 2 above)
Comment (roman-khimov) [reply pkg/wallet/wallet_test.go]: And it no longer is a `Path` probably, just a template.

**Pull request merged**